engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -576,6 +576,11 @@ public void startApplication() throws YarnException, IOException
         vargs.add("-Dlog4j.debug=true");
       }
 
+      String loggerAppender = dag.getValue(LogicalPlan.LOGGER_APPENDER);
```

Comment:
Do I need to specify the LOGGER_APPENDER twice with -Dapex.logger.appender and -Ddt.attr.LOGGER_APPENDER if I wanted the appender to be used for both client and app? If I just specify apex.logger.appender can it also set the LOGGER_APPENDER attribute if it is not present.